src/lib/homestore_backend/hs_blob_manager.cpp

```diff
@@ -381,6 +386,11 @@ BlobManager::NullAsyncResult HSHomeObject::_del_blob(ShardInfo const& shard, blo
         return folly::makeUnexpected(BlobError(BlobErrorCode::NOT_LEADER, repl_dev->get_leader_id()));
     }
 
+    if (!repl_dev->is_ready_for_traffic()) {
```

Comment:
Do we need to add this check also for `_get_blob`? As `RETRY_REQUEST` can be a clear hint for client to retry.

Reply:
I think we need this for get_blob as well